### ISSUE TITLE
Use Relative Paths rather than Absolute paths

### ISSUE THREAD

**Issue by Icedude907** [AGS-Programming-Group/Ags-Programming-Group.github.io#15]
Just a quick thing from a random child.
I noticed that in some the html files some links are absolute eg: `https://ags-programming-group.github.io/button.css`. This should be changed to `/button.css` to allow editing of files that may have been removed, allow editing offline and on the whole just make it easier to work with.
Not a insanely important matter, but I will probably fix it if I start to work on this project with a pull request.

From your friendly neighbourhood weird little kid
_Icedude907_

**Comment by Yamboy1** [comment]
Yeah i thought about this, then decided to make most of the links absolute

**Comment by Yamboy1** [comment]
but then i guess it's a good idea to change them back

**Comment by Yamboy1** [comment]
the only thing is that if im not on a webserver, it stuffs up and thinks i want to access a file in the root directory of my computer @Icedude907

**Comment by Yamboy1** [comment]
what do you think @blazingforest

**Comment by blazingforest** [comment]
Well, the absolute path will always work, but idk whatever you want to do
